qemu/tests/cpu_add.py

```diff
@@ -138,8 +140,9 @@ def onoff_para_opt(onoff_params):
     vcpu_been_pluged = 0
     for i in range(vcpu_need_hotplug):
         hotplug_vcpu_params = params.object_params("hotplug_vcpu%s" % i)
-        plug_cpu_id = len(vm.vcpu_threads)
-        plug_cpu_id = hotplug_vcpu_params.get("id", plug_cpu_id)
+        plug_cpu_id = hotplug_vcpu_params.get("id")
```

Comment:
If "id" is not defined, will get wrong value of "id".

15:22:52 DEBUG| The error info is:
 'QMP command 'cpu-add' failed    (arguments: {'id': 'smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.cpu_add_test.one_cpu'},    error message: {u'class': u'GenericError', u'desc': u"Invalid parameter type for 'id', expected: int"})'